src/asm2wasm.h

```diff
@@ -1286,6 +1285,7 @@ void Asm2WasmBuilder::processAsm(Ref ast) {
   };
 
   PassRunner passRunner(&wasm);
```

Comment:
Can use the second passRunner constructor, which receives passOptions as a second param. That would copy in the features.

Reply:
Wouldn't that also copy in OptLevel and ShrinkLevel, which we don't want?

Reply:
Good point, yeah.